confd/pkg/backends/calico/client.go

```diff
@@ -1349,6 +1350,15 @@ func (c *client) getNodeMeshPasswordKVPair(v3res *apiv3.BGPConfiguration, key in
 	}
 }
 
+func (c *client) getIgnoredInterfacesKVPair(v3res *apiv3.BGPConfiguration, key interface{}) {
+	ignoredInterfaces := getBGPConfigKey("ignored_interfaces", key)
```

Comment:
nit: might make this easier to follow if the variable was something like `ignoredIfacesKey` or `ignoredInterfacesKey`